evm/src/executor/builder.rs

```diff
@@ -48,6 +49,17 @@ impl ExecutorBuilder {
         self
     }
 
+    /// Enables the fuzzer for data collection and maybe call overriding
+    #[must_use]
+    pub fn with_fuzzer(
```

Comment:
I think we should probably rename this to something else - it's a little bit ambigious that we now have to "fuzzer" concepts, and one of them wraps the executor (and lives outside of `ExecutorBuilder`) and a different one is an inspector (and lives inside of `ExecutorBuilder`)

Reply:
I agree, especially since it also does data collection, and the actual fuzzing option is optional. I'm open to suggestions... :sweat_smile: